### ISSUE TITLE
Provide access to the path in `is/createDataType`

### ISSUE THREAD

**Issue by andrefgneves** [TexteaInc/json-viewer#106]
Hi 👋🏻

I'd like to know the path of the `value` on my `valueTypes`. Something like:

```
const valueTypes = [
  {
    is: (value, path) => path.pop() === 'some name',
    Component: () => ...,
  },
  createDataType(
    (value, path) => path.includes('some name'),
    () => ...
  ),
]
```

**Comment by Himself65** [comment]
that's a good idea

**Comment by andrefgneves** [comment]
Thank you!

**Comment by Himself65** [comment]
Landed in 2.10.0
